package-lock.json

```diff
@@ -3361,6 +3362,15 @@
         "tslib": "^2.4.0"
       }
     },
+    "node_modules/@types/cypress": {
```

Comment:
can you revert the `package-lock.json` (or regenerate with `npm i`) now you have removed `@types/cypress`?